incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,20 @@
+//////////  S V G   M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
+
+// Element mixin to be used for general background images
+// as you would use a retina mixin
+@mixin svg-element($file, $width: 100%, $height: $width)
+    @include background(image-url('svg/#{$file}.svg'))
```

Comment:
I don't think we need to use the Compass mixin for this as its just a url. Have you seen any particular output from this other than `background-image: url(images/svg/my-shape.svg)`?